chpl/chpl-service/src/main/resources/environment.properties.template

```diff
@@ -7,9 +7,15 @@ basePath=/rest
 questionableActivityEmail=theonc.certification@hhs.gov;onc_chpl@hhs.gov
 questionableActivityThresholdDays=0
 maxActivityRangeInDays=60
+
+#cache properties
 enableCacheInitialization=true
 cacheClearTimeoutSecs=15
 cacheInitializeTimeoutSecs=300
+#how long to wait before first loading the prefetched listing cache
+listingCacheRefreshInitialDelayMillis=300000

```

Comment:
Got it. That sounds like a plan.

Please add these properties into the build files 😄